pwiz_aux/msrc/utility/vendor_api/ABI/WiffFile2.ipp

```diff
@@ -831,8 +840,7 @@ void WiffFile2Impl::setSample(int sample) const
             IList<ISample^>^ unwrappedAllSamples = allSamples;
             this->msSample = (ISample^)unwrappedAllSamples[sample - 1];
 
-            auto experimentRequest = DataReader()->RequestFactory->CreateExperimentsReadRequest();
-            experimentRequest->SampleId = msSample->Id;
+            auto experimentRequest = DataReader()->RequestFactory->CreateExperimentsReadRequest(msSample->Id, true);
```

Comment:
What does the true parameter do?

Reply:
It retrieves detailed experiment info, such as fragmentation type and electron kinetic energy.
This line should be called only once per sample, so I think the impact to performance will be very small.

IExperimentsReadRequest CreateExperimentsReadRequest(
	string sampleId,
	bool readDetailedParameters
)